content/general/concepts/aaron-swartz/aaron-swartz.md

```diff
@@ -12,7 +12,9 @@ CatalogContent:
   - 'paths/code-foundations'
 ---
 
-Aaron Swartz (1986-2013) was an American programmer, writer, entrepreneur, and internet activist. He was involved in the development of the RSS web feed format, Markdown language, Creative Commons organization, and Python web framework web.py.
+![Aaron Swartz](https://raw.githubusercontent.com/Codecademy/docs/main/media/aaron_swartz.png)
+
+**Aaron Swartz** (1986-2013) was an American programmer, writer, entrepreneur, and internet activist. He was involved in the development of the RSS web feed format, [Markdown](https://www.codecademy.com/resources/docs/markdown) language, [Creative Commons licenses](https://creativecommons.org/), and Python web framework [web.py](https://webpy.org/).
 
 He was a co-founder of a startup called Infogami, which later became the social news site Reddit. In his later years, he co-founded an activist organization, Demand Progress, and led the charge fighting the Stop Online Piracy Act (SOPA).
 
```

Comment:
while we're here can we fix line 29 (or 27 in the original)? I think it's supposed to say "mass download" not "massive download".